conf/defaults.ini

```diff
@@ -174,6 +174,9 @@ disable_gravatar = false
 # data source proxy whitelist (ip_or_domain:port separated by spaces)
 data_source_proxy_whitelist =
 
+# disable protection against brute force (frequent) login attempts
+disable_login_attempts_validation = false
```

Comment:
I don't know if this a good name

Reply:
`disable_brute_force_login_protection` seems more clear

Reply:
Done. Thanks